packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -749,6 +748,30 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregate qualities: algorithm</name>
+      <type>string</type>
+      <default>forward</default>
+      <description>Algorithm to use ``forward'' or ``reverse.''</description>
```

Comment:
Maybe a quick explanation about what `Algorithm` refers to?

Reply:
Fixed